README.md

```diff
@@ -706,11 +706,12 @@ and CloudWatch shows logs entries in the `/aws/apigateway/…` log group but not
 [KMSAccessDeniedException] error. In the AWS Console, go to the Lambda function
 details page, click on the `Test` tab, and click on the `Test` buttton. 
 
-Note that is normal for some Lambda to fail the test due to the parameters of
-the test event. Examine the error message to determine if the failure is due to
-a KMSAccessDeniedException which would be explicitly specified.
+Note that is normal for some Lambda functions to fail the test due to the
+parameters of the test event. Examine the error message to determine if the
+failure is due to a `KMSAccessDeniedException` which would be explicitly
+specified.
 
-If a KMSAccessDeniedException is confirmed, with the target deployment
+If a `KMSAccessDeniedException` is confirmed, with the target deployment
```

Comment:
"If `KMSAccessDeniedException` is confirmed to be the cause"

comma between selected and run. But still awkward. Please improve.